CONTRIBUTING.md

```diff
@@ -86,6 +86,32 @@ Follow these steps to set up your local development environment on Unix/Mac OS:
 </details>
 
 
+## Naming Patterns
+
+### Branch name
+```
+(feat|fix|docs|style|refactor|test|revert)/taskId_task-short-description
+Ex.: feat/1_add-commit-and-branch-name-styles-to-readme
+```
+
+### Commit name
+```
+(feat|fix|docs|style|refactor|test|revert): update descriptions
+Ex.: bug: update readme
+```
+
+### Commit description
+```
+tickets: taskId, taskId
+Ex.: tickets: 1, 2, 3
+```
+
+### Pull-request name
+```
+(Feat|Fix|Docs|Style|Refactor|Test|Revert)/taskId description
+Ex.: Fix/15 unnecessary request removed
```

Comment:
Again: follow GitHub conventions for issue referencing, e.g. `#15`

Reply:
Fixed